.github/actions/templates/avm-validateModuleDeployment/action.yml

```
@@ -55,14 +55,47 @@ inputs:
   customLocation:
     description: "Custom location overwrite, if needed"
     required: false
+  e2eignore:
+    description: "A test case will not be deployed if an .e2eignore file exists"
+    required: false
 
 runs:
   using: "composite"
   steps:
+    # [Deployment validation] task(s)
+    # -------------------------------
+    - name: "Validate Test Execution"
+      uses: azure/powershell@v2
+      with:
+        azPSVersion: "latest"
+        errorActionPreference: "SilentlyContinue"
+        inlineScript: |
+          # the first step is to check, if the deployment should be bypassed
+
+          # Resolve template file path
+          $moduleTestFilePath = Join-Path $env:GITHUB_WORKSPACE '${{ inputs.templateFilePath }}'
+
+          # Check if the deployment test should be bypassed
+          if (-not [String]::IsNullOrEmpty('${{ inputs.e2eignore }}') -and '${{ inputs.e2eignore }}' -eq 'true') {
```

```suggestion
          if ('${{ inputs.e2eIgnore }}' -eq 'true') {
```
First check is redundant as the default `''` will anyways be comparable against 'true'.